### PR TITLE
FLUT-6317-[documentation][flutter]: Hide Rows And Columns - XlsIO

### PR DESCRIPTION
**Pull request by KonduruKeerthi**
Hi @johnsonmonohar ,

Kindly review the changes and let me know your feedback.

Regards,
Keerthi.

### CODE REVIEW

File: Flutter/xlsio/worksheet-rows-and-columns-manipulation.md [hunk 1]
```diff
@@ -183,6 +183,53 @@ workbook.dispose();
 
 {% endhighlight %}
 
+## Show or Hide Rows and Columns
 
+Visibility of rows and columns can be set by using the showRows and showColumns methods as shown below.
```
Comment (johnsonmonohar) [inline Flutter/xlsio/worksheet-rows-and-columns-manipulation.md:188]: Don't we need to use like this for technical words or APIs? - `showRows`

**Review by johnsonmonohar — COMMENTED** [review]
Please address the feedback.

File: Flutter/xlsio/worksheet-rows-and-columns-manipulation.md (hunk 1 above)
Comment (KonduruKeerthi) [reply Flutter/xlsio/worksheet-rows-and-columns-manipulation.md]: Yes sir. We can highlight the APIs or API reference links. I have added the API reference links to all the 3 APIs

**Review by johnsonmonohar — APPROVED** [review]
The changes are approved.

**Pull request merged**